README.md

```diff
@@ -220,15 +216,58 @@ pm2 start --name lisk app.js -- -p [port] -a [address] -c [config-path] -n [netw
+
+| Option             | ENV Variable           | Config Option   | Description                                                                                                                                                                       |
+| ------------------ | ---------------------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --network<br> -n   | LISK_NETWORK           |                 | Which configurations set to use, associated to lisk networks. Any of this option can be used `devnet`, `alphanet`, `betanet`, `testnet` and `mainnet`. Default value is `devnet`. |
+| --config<br> -c    | LISK_CONFIG_FILE       |                 | Path the custom configuration file, which will override values of `config/default/config.json`                                                                                    |
+| --port<br> -p      | LISK_WS_PORT           | wsPort          | TCP port for P2P layer                                                                                                                                                            |
+| --http-port<br> -h | LISK_HTTP_PORT         | httpPort        | TCP port for HTTP API                                                                                                                                                             |
+| --address<br> -a   |                        | address         | Listening host name or ip                                                                                                                                                         |
+| --database<br> -d  |                        | db.database     | PostgreSQL database name to connect                                                                                                                                               |
+| --peers<br> -p     | LISK_PEERS             | peers.list      | Comma separated list of peers to connect in the format `192.168.99.100:5000,172.169.99.77:5000`                                                                                   |
+| --log<br> -l       | LISK_CONSOLE_LOG_LEVEL | consoleLogLevel | Log level for lisk for console output                                                                                                                                             |
+| --snapshot<br> -s  |                        |                 | Number of round for which take the snapshot. If none specified it will use the highest round available.                                                                           |
+
+For more detail understanding of configuration read this [online documentation](https://lisk.io/documentation/lisk-core/user-guide/configuration)
+
+### Examples
```

Comment:
What about the examples?

Reply:
This sub-heading is under section `Configuring Lisk`, so we add more common use cases for configuration under Examples.